src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -40,6 +40,7 @@
       "type": "build",
       "version": "1.1.0-*"
     },
+    "Microsoft.Extensions.Logging": "1.1.0-preview2-22526",
```

Comment:
My apologies @jbagga for making a poor suggestion.

@Eilon we have some `[Obsolete]` members in MVC now that don't match the pattern you described. The [old constructor](https://github.com/aspnet/Mvc/blob/4e1ec39a1f1d15a4ec74312809e3233bc106b77b/src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/DefaultHtmlGenerator.cs#L60-L61) in `DefaultHtmlGenerator` works fine (though we definitely should remove it and the unnecessary parameters in the new constructor in 2.0). Worse, the [old constructor](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Abstractions/Formatters/OutputFormatterCanWriteContext.cs#L24) in `OutputFormatterCanWriteContext` will likely cause NREs in MVC if used. Worth a bug to clean some of this up?